src/EXTRA-COMPUTE/compute_stress_mop.cpp

```diff
@@ -237,46 +246,47 @@ void ComputeStressMop::init()
   if (force->pair->single_enable == 0)
     error->all(FLERR, "Pair style does not support compute stress/mop");
 
-  // Errors
+  // issue an error for unimplemented bond/angle/dihedral potentials
 
-  if (comm->me == 0) {
+  if (bondflag == 1 && !(force->bond)) {
+    bondflag = 0;
+  }
 
-    // issue an error for unimplemented intramolecular potentials or Kspace.
+  if (angleflag == 1 && !(force->angle)) {
+    angleflag = 0;
+  }
 
-    if (force->bond) {
-      bondflag = 1;
-      if (comm->nprocs > 1)
-        error->one(FLERR, "compute stress/mop with bonds does not (yet) support MPI parallel runs");
-    }
+  if (dihedralflag == 1 && !(force->dihedral)) {
+    dihedralflag = 0;
+  }
 
-    if (force->angle) {
-      if (force->angle->born_matrix_enable == 0) {
+  if (angleflag == 1 && force->angle) {
+    if (force->angle->born_matrix_enable == 0) {
```

Comment:
I don't understand why the born_matrix_enable flag comes into play here. Edit: My bad, I skipped its use in the code.